### ISSUE TITLE
For proxy_socket read or write of unistd not work.

### ISSUE THREAD

**Issue by engineer1109** [emscripten-core/emscripten#16513]
send(sock, message, strlen(message), 0) works

write(sock, message, strlen(message)) no response

**Comment by sbc100** [comment]
Indeed, I believe the way that PROXY_POSIX_SOCKETS is the it overrides certain syscalls, and `read` and `write` are not among them.   See system/lib/websocket/websocket_to_posix_socket.c:

https://github.com/emscripten-core/emscripten/blob/6abf9a31795cc96023ed5213b15f3c90416409a7/system/lib/websocket/websocket_to_posix_socket.c#L544

There is no integration with open open/close/read/write or the reset of the filesystem layer as far as I can tell.   Doing such an integration could be fair amount of work.
